server/config/urls.py

```diff
@@ -18,4 +19,5 @@
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
+    url(r'^index/', TemplateView.as_view(template_name='index.html')),
```

Comment:
@ordavidil which existing url would you suggest to use? I only see ^api/ and ^admin/ in master